.gitlab-ci.yml

```diff
@@ -229,6 +247,18 @@ build:bleeding-edge:
     EXTRA_CONF: "-native-compiler yes -coqide opt"
     EXTRA_PACKAGES: "$COQIDE_PACKAGES"
 
+windows64:
+  <<: *windows-template
+  variables:
+    ARCH: "64"
+    SETUP: "setup_x86-64.exe"
```

Comment:
SETUP is reset in the gitlab.bat so not needed here.

Reply:
It's now used in `.gitlab-ci.yml`. I should probably remove it from `gitlab.bat`.

Reply:
Why not include the download in the .bat?

Reply:
How?

Reply:
No, the part that performs the download is PowerShell.

Reply:
Can powershell not be called from bat?

Reply:
I have no idea, I know nothing about Windows dev.

Reply:
But anyway, this download is setting up dependencies, so I'm not shocked by seeing it in `.gitlab-ci.yml`. Isn't it what we do for other builds?